### DISCUSSION TITLE
PS4 | PS5 PlayStation Youtube приложение

### DISCUSSION THREAD

**Discussion by UchihaSV** [bol-van/zapret#423] [Q&A]
Дарова, накатил OpenWRT + Zapret с вашими конфигами из дискуссий и заработал ютуб, но на PS4 приставке там свое приложение для Youtube видео и там при его запуске выдает ошибку "Нет интернета", хотя инет есть и программное обеспечение обновляется и другое приложение Twitch работает. Если зайти через встроенный браузер плойки на youtube.com он загружается нормально, может нужно дополнительно что-то прописать именно под это видео приложение ютуба на сони?
![hq720](https://github.com/user-attachments/assets/4fda6959-57d5-4449-a126-dca35169aa59)

**Comment by Zus-dlr** [comment]
Попробуйте установить zapret с QUIC и, соответственно, его настроить. Всё должно заработать.

**Reply by egorsey** [reply]
Есть возможность расписать подробнее?

**Reply by HentayHentay** [reply]
> zapret с QUIC

запрет quic понимает
из коробки curl часто не может quic дернуть, без пересборки/модификации.
